src/libspark/coin.cpp

```diff
@@ -56,9 +56,9 @@ Coin::Coin(
 	// Type-specific elements
 	//
 
+	this->v = v;
+	this->address = address;
```

Comment:
I am not agree to do this change, Very risky to add v and address into Spark::Coin, even if we don't serailize them,  wallet knows this data inside CSparkMintMeta, please use it to get this data, and revert this two lines.